spec/acceptance/test_spec.rb

```diff
@@ -38,8 +37,8 @@ class {'satellite_pe_tools':
       Helper.instance.run_shell('puppet agent -t', expect_failures: true)
     end
 
-    it 'contains the report text in Satellite' do
-      expect(satellite_get_last_report(satellite_host, server_host).to_s).to match(%r{This is a test from Puppet to Satellite})
+    xit 'contains the report text in Satellite' do
```

Comment:
typo
```suggestion
    it 'contains the report text in Satellite' do
```

Reply:
its intentional change, I have commented this change as I mentioned on slack.
Not sure if you have idea but this is how you can move the spec into pending.

Reply:
AH! we would also opt for this way, so we can add a reason why also.
https://github.com/puppetlabs/puppetlabs-node_encrypt/blob/ab6e46cd6fc9c14a3297df89b2ec690e026828ed/spec/defines/file_spec.rb#L38